downstream/assemblies/hub/assembly-populate-container-registry.adoc

```diff
@@ -12,6 +12,24 @@ ifdef::context[:parent-context: {context}]
 [role="_abstract"]
 By default, private {HubName} does not include container images. To populate your container registry, you need to push a container image to it. The procedures in this section describe how to pull images from the Red Hat Ecosystem Catalog (registry.redhat.io), tag them, and push them to your private {HubName} container registry.
 
+[IMPORTANT]
+====
+Image manifests and filesystem blobs are served directly from `registry.redhat.io` and `registry.access.redhat.com`. 
+However, from 1st of May 2023, filesystem blobs are served from `quay.io` instead. 
+To avoid problems pulling container images, you must enable outbound connections to the following hostnames:
+
+* `cdn.quay.io`
+* `cdn01.quay.io`
+* `cdn02.quay.io`
+* `cdn03.quay.io`
+
+This change should be made to any firewall configuration that specifically enables outbound connections to `registry.redhat.io` or `registry.access.redhat.com`.
+
+Use the hostnames instead of IP addresses when configuring firewall rules. 
+
+After making this change you can continue to pull images from `registry.redhat.io` and `registry.access.redhat.com`. You do not require a `quay.io` login, or need to interact with the `quay.io` registry directly in any way to continue pulling Red Hat container images.
```

Comment:
Question: Should the sentence that begins with "You do not require a `quay.io` login" actually mean "You do not  need a `quay.io` login" or "You are not required to have a `quay.io` login"? 

I'll let you make that decision. Just reads slightly unusual when compared to other statements of requirements from the user perspective.